examples/vue/nuxt/package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "algoliasearch": "4.14.3",
     "cross-env": "^5.2.0",
+    "css-loader": "^4.3.0",
```

Comment:
Without this line, the example doesn't run (probably due to it being in a monorepo now). I set the version to one that is already in yarn.lock.